test/test-packages.py

```diff
@@ -19,17 +19,21 @@ def main():
         packages = yaml.safe_load(f.read())
 
     def get_test_set():
+        installers = {'CONDA_NAME': 'container-conda-test.sh', 'PIP_NAME': 'container-script.sh'} 
         for package in packages['packages']:
-            package_main_name = re.findall(r'([\S]+)', package['PIP_NAME'])[0]
-            package_list = package['PIP_NAME']
-            package['main_name'] = package_main_name
-            py_script = package['PKG_TEST']
-            for container in ['amazon-linux2', 'centos8', 'centos8-py38', 'focal']:
-                yield (package_main_name, package_list, container, 'container-script.sh', py_script, container)
-            if 'APT_NAME' in package:
-                yield (package_main_name, package['APT_NAME'], 'focal', 'container-apt-test.sh', py_script, 'focal-apt')
-            if 'YUM_NAME' in package:
-                yield (package_main_name, package['YUM_NAME'], 'centos8', 'container-yum-test.sh', py_script, 'centos8-yum')
+            for install_type in installers.keys():
+                if install_type not in package: 
+                    continue
```

Comment:
Won't this skip all the apt and yum tests?

Reply:
yes I need to revert it back how it was earlier

Reply:
@geoffreyblake actually it won't skip the yum and apt tests as even in current wheel tester version, we consider there always exists 'PIP_NAME' in packages, considering this the apt and yum tests wont be skipped. 
But we shouldn't assume this

Reply:
That might be true now, but it could cause a bug in the future. You could fix it by adding the two new keys to the installers dict and updating the yield lines to use the same dict lookup for the script.

Reply:
Yes I am adding those changes, just testing them locally

Reply:
I am thinking to add `PKG_NAME` key to packages.yaml that will be used to generate report